doc/article/en-US/angular1-to-aurelia.md

```diff
@@ -0,0 +1,526 @@
+
+<code-listing heading="Angular Interpolation">
+	<source-code lang="HTML">
+		<div id="hello">Hello, my name is {{name}}!</div>
+	</source-code>
+</code-listing>
+
+Aurelia makes use of ES6 template strings in its templates, so the Aurelia version becomes this:
+
+<code-listing heading="Aurelia Interpolation">
+	<source-code lang="HTML">
+		<div id="hello">Hello, my name is ${name}!</div>
+	</source-code>
+</code-listing>
+
+Aurelia's template strings use pipe syntax for ValueConverters (filters in Angular).
+
+#### Template behaviors
+
+`ng-show` becomes `show.bind`. `ng-if` becomes `if.bind`. `ng-click` becomes `click.delegate`.
+
+Angular's repeat syntax looks like this:
+
+<code-listing heading="Angular Interpolation">
+	<source-code lang="HTML">
+		<ul data-ng-repeat="item in items">
+			<li id="{{item.id}}">{{item.name}}</li>
+		</ul>
+	</source-code>
+</code-listing>
+
+In Aurelia, the repeat is not on the parent element. It looks like this:
+
+<code-listing heading="Aurelia Interpolation">
+	<source-code lang="HTML">
+		<ul>
+			<li repeat.for="item of items" id="${item.id}">${item.name}</li>
+		</ul>
+	</source-code>
+</code-listing>
+
+Event binding is also a bit different:
+
+<code-listing heading="Angular Event Binding">
+	<source-code lang="HTML">
+		<button data-ng-click="myFunction(argument)">myFunction button</button>
+	</source-code>
+</code-listing>
+
+<code-listing heading="Aurelia Event Binding">
+	<source-code lang="HTML">
+		<button click.delegate="myFunction(argument)">myFunction button</button>
+	</source-code>
+</code-listing>
+
+#### Converting `ng-include` to `<compose>`
+
+<code-listing heading="Angular View Composition">
+	<source-code lang="HTML">
+		<div data-ng-include="myView"></div>
+	</source-code>
+</code-listing>
+
+<code-listing heading="Aurelia View Composition">
+	<source-code lang="HTML">
+		<compose view-model="myViewModel" view="myView"></compose>
+	</source-code>
+</code-listing>
+
+### Converting Angular directives to Custom Elements and Attributes
+
+Finally, the meat and potatoes - it's time to migrate our Angular directives.
+
+Angular and Aurelia both support the concept of extending HTML by creating custom elements and attributes. However, Aurelia handles the creation of these in a different way.
+
+#### Converting a directive to an Aurelia custom element
+
+Creating a custom element in Angular required us to write an Angular directive; we would return a configuration object that told Angular how to create our object through object properties including `scope`, `restrict`, `template`, `templateUrl`, and `link`.
+
+Instead of a configuration object, Aurelia takes a class instance.
+
+An Angular directive may have looked like this:
+
+<code-listing heading="Angular Custom Components">
+	<source-code lang="ES 2015/2016">
+		angular.directive('display', ['$window', '$document', 'animate', function ($window, $document, animate) {
```

Comment:
I think there should be several examples of Angular element type directives to explain different ways that Angular directives could have been used and how to map them to Aurelia Custom element:
For example scope - isolated or not. For the first example I think the directive should have isolated scope (and probably accept different types of arguments from the caller, including primitives, objects, function callbacks with arguments)